### ISSUE TITLE
[Suggestion] How to make Bees and actual viable option

### ISSUE THREAD

**Issue by tjsd** [TPPIDev/TestPackPleaseIgnore2#211]
TLDR; I would like to remove binnie's genetics and read Gendustry

Gendustry:
It is overpowered and to easy to use, so by changing a core component in the gendustry machines it would make the mod more balanced.
This will require you to breed all the way to Imperial line and Industrious line, and you would have to grab enough honey to make a alveary block.

http://i.imgur.com/tCNyXuz.png < breeding of bees

http://ftbwiki.org/Bee_Receptacle < main component in all gendustry machines

The new Bee Receptacle recipe would require Royal Jelly, Pollen and an Alveary Block
http://i.imgur.com/e4HBn9q.png < suggestion for new tppi tweaks block to balance the mod.
http://ftbwiki.org/Pollen
http://ftbwiki.org/Royal_Jelly
http://ftbwiki.org/Alveary

My main problem with Binnie's Genetics is the lack of documentation and updates, the issues raised and not dealt with is quite huge (https://bitbucket.org/binnie567/binnie-mods/issues).
The process of making the liquids for the mod to work at all is rather convulted and tedious at best.

**Comment by tjsd** [comment]
I ran with my "fix" on a custom pack for a while, and I feel it works out pretty well. The normal breeding required to get to gendustry gates it pretty nice. Here is the minetweaker script for the change if you decide to implement it someday with tppi or minetweaker.

// Gendustry Tweak

val beeframe = <gendustry:BeeReceptacle:0>;

//
// List of Materials used
//
val royaljelly = <ore:dropRoyalJelly>;
val pollen = <ore:itemPollen>;
val alvblock = <Forestry:alveary>;
val goldblock = <minecraft:gold_block>;
val redstone = <ore:dustRedstone>;

// Modify Bee Receptacle
recipes.removeShaped(beeframe);
recipes.addShaped(beeframe, [[royaljelly, royaljelly, royaljelly], [pollen, goldblock, pollen], [redstone, alvblock, redstone]]);

**Comment by tjsd** [comment]
You could also remove the advanced mutatron and only leave the basic one. So you still have a random chance for outcome instead of picking the mutation you want between two bees.

http://ftbwiki.org/Advanced_Mutatron

**Comment by Euphoreich** [comment]
If anything I'd change the labware recipe to only make 1 labware instead of 16, and take 2 hardened glass/fused quartz instead of glass panes. Labware is used for doing anything interesting in advanced genetics, and increasing the price is a great way to nerf the mod overall. This will mean the player will need ample diamonds and some industry setup.
